ownership-chain/runtime/src/tests/spec_version.rs

```diff
@@ -0,0 +1,108 @@
+use crate::VERSION;
+use std::{
+	io::{Error, ErrorKind},
+	num::ParseIntError,
+};
+
+/// Converts a version string in the format "major.minor.patch" to an integer.
+/// Each part of the version (major, minor, patch) must be a number less than 100.
+/// The resulting integer is in the format major*10000 + minor*100 + patch.
+///
+/// # Arguments
+/// * `version` - A string slice representing the version.
+///
+/// # Returns
+/// A `Result` containing either the integer representation of the version
+/// or an `Error` if the input format is incorrect or any part of the version
+/// is 100 or greater.
+///
+/// # Examples
+/// ```
+/// let version = "1.2.3";
+/// let int_version = version_to_int(version).unwrap();
+/// assert_eq!(int_version, 10203);
+/// ```
+fn version_to_int(version: &str) -> Result<u32, Error> {
```

Comment:
I suggest to find a better name that gives more context of its purpose. (e.g.: `parse_cargo_pkg_version_to_runtime`). Also it is not needed to include the returned value `int` as part of the name.

Reply:
I don't agree in the `cargo_pkg` part cause it is not what the function does.
maybe `convert_version_to_number` might be better